sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/Attachment.kt

```diff
@@ -17,7 +17,7 @@ actual class Attachment : IAttachment {
         get() = cocoaAttachment.data?.toByteArray()
 
     actual override val contentType: String?
-        get() = cocoaAttachment.contentType
+        get() = cocoaAttachment.contentType ?: "application/octet-stream"
```

Comment:
Is there a reason to fallback? if the field is nullable and the server has its own default

Reply:
If it's enforced in the server then it's not needed. I changed it because my attachment tests were failing but I'll adjust those as well

Reply:
I dont think that would be the problem, but double-check if its still failing.